spec/reek/smells/nil_check_spec.rb

```diff
@@ -19,55 +19,69 @@ def nilcheck foo
     expect('def no_nils; end').not_to reek_of(:NilCheck)
   end
 
-  it 'reports when scope uses multiple nil? methods' do
+  it 'reports when scope use #nil?' do
     src = <<-EOS
     def chk_multi_nil(para)
       para.nil?
-      puts "Hello"
-      \"\".nil?
     end
     EOS
     expect(src).to reek_of(:NilCheck)
   end
 
-  it 'reports twice when scope uses == nil and === nil' do
+  it 'reports when scope uses == nil' do
     src = <<-EOS
     def chk_eq_nil(para)
       para == nil
+    end
+    EOS
+    expect(src).to reek_of(:NilCheck)
+  end
+
+  it 'reports when scope uses === nil' do
+    src = <<-EOS
+    def chk_eq_nil(para)
       para === nil
     end
     EOS
     expect(src).to reek_of(:NilCheck)
   end
 
   it 'reports when scope uses nil ==' do
-    expect('def chk_eq_nil_rev(para); nil == para; end').to reek_of(:NilCheck)
+    src = <<-EOS
+    def chk_eq_nil_rev(para)
```

Comment:
Can we rename this as well and above? The name decreases readability and hints at being relevant when it isn't.
How about just "m"?